modules/about-openshift-gitops.adoc

```diff
@@ -0,0 +1,10 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/working-with-openshift-gitops.adoc
+
+[id="about-openshift-gitops_{context}"]
+= About OpenShift GitOps
+
+Red Hat OpenShift GitOps is a declarative way to implement continuous deployment for cloud knative appliactions. Use Red Hat OpenShift GitOps to create repeatable processes for managing OpenShift Container Platform clusters and applications across multi-cluster Kubernetes environments. Red Hat OpenShift GitOps handles and automates complex deployments at a fast pace, saving time during deployment and release cycles. The Red Hat OpenShift GitOps workflow pushes an application through all its stages, development, testing, staging, and production. Red Hat OpenShift GitOps either deploys a new application or updates an existing one, so you only need to update the repository. Red Hat OpenShift GitOps handles and automates everything else.
+
+Red Hat OpenShift GitOps is a declarative way to implement continuous deployment for cloud knative applications. Red Hat OpenShift GitOps has a Git repository that contains a declarative description of the infrastructure you need in your specified environment. This Git repository also contains an automated process to make your environment match the described state.
```

Comment:
Remove: Red Hat OpenShift GitOps is a declarative way to implement continuous deployment for cloud knative applications. since you added it above